packages/fetchai/skills/confirmation_aw1/README.md

```diff
@@ -0,0 +1,16 @@
+# Confirmation AW1
+
+## Description
+
+The confirmation_aw1 skill is a skill for handling registrations in Agent World 1.
```

Comment:
The confirmation_aw1 skill is for handling registrations in Agent World 1.

Reply:
see below